File: discover_ruby/manipulate_the_data.rb
require 'HTTPClient'
require 'json'

extheaders = {
 'User-Agent' => 'Holberton_School',
 'Authorization' => 'token 39fed9cadb8579194e82b4bc4aefa34b8fab30b7'
}


clnt = HTTPClient.new
res = clnt.get_content('https://api.github.com/search/repositories?q=language:ruby&sort=stars&order=desc', query = nil, extheaders = {})
json_parse = JSON.parse(res)
array = json_parse["items"]
#puts json_parse["items"][0]["full_name"]
join_array = array.map {|a| a["full_name"]}
puts join_array.join("\n")

File: discover_more_python/manipulate_the_data.py
import urllib2
import urllib
import json

request_headers = {
  'User-Agent': 'Holberton_School',
  'Authorization': 'token aa691ecddbb46f017092bdb87a3c55300bd33e9d'
}

url = 'https://api.github.com/search/repositories?q=language:python&sort=stars&order=desc'
req = urllib2.Request(url, None, request_headers)
response = urllib2.urlopen(req)
the_page = response.read()
parsed_json = json.loads(the_page)

for i in parsed_json['items']:
	print i['full_name'] 





